app/src/main/java/com/imashnake/animite/features/home/Home.kt

```diff
@@ -48,94 +48,108 @@ fun Home(
+                                        )
                                     )
-                                ) {
-                                    launchSingleTop = true
-                                }
-                            }
-                        )
-
-                        Spacer(modifier = Modifier.size(24.dp))
+                                )
+                                .fillMaxWidth()
+                                .aspectRatio(1f)
+                        ) {  }
 
                         Text(
-                            text = "Popular This Season",
-                            color = Text,
-                            fontSize = 14.sp,
-                            fontFamily = manropeFamily,
-                            fontWeight = FontWeight.Bold,
-                            modifier = Modifier.padding(start = 24.dp)
+                            text = "おかえり",
+                            color = MaterialTheme.colorScheme.onSecondaryContainer,
+                            style = MaterialTheme.typography.displayLarge,
+                            modifier = Modifier
+                                .align(Alignment.BottomStart)
+                                .padding(start = 24.dp, bottom = (24 + 18).dp)
                         )
+                    }
 
-                        Spacer(modifier = Modifier.size(12.dp))
+                    // TODO: Use `verticalArrangement` instead of the `Spacer`s.
+                    Column {
+                        Spacer(
+                            modifier = Modifier.size((LocalConfiguration.current.screenWidthDp - 18).dp)
+                        )
 
-                        MediaSmallRow(
-                            mediaList = popularThisSeasonMediaList,
-                            onItemClick = { itemId ->
-                                navigator.navigate(
-                                    MediaPageDestination(
-                                        id = itemId,
-                                        mediaTypeArg = homeMediaType.rawValue
-                                    )
-                                ) {
-                                    launchSingleTop = true
+                        Column(
+                            modifier = Modifier
+                                .clip(backdropShape)
+                                .background(MaterialTheme.colorScheme.background)
+                        ) {
+                            Spacer(modifier = Modifier.size(24.dp))
+
+                            Text(
+                                text = "Trending Now",
+                                color = MaterialTheme.colorScheme.onBackground,
+                                style = MaterialTheme.typography.headlineMedium,
+                                modifier = Modifier.padding(start = 24.dp)
+                            )
+
+                            Spacer(modifier = Modifier.size(12.dp))
```

Comment:
Spacers for 12dp and 24dp? Can these be decorators? Or `fun SmallSpacer()` | `fun MediumSpacer()` which can be defined as 12dp/24dp